index.html

```diff
@@ -0,0 +1,484 @@
+            <div class="first-level-hover-wrapper">
+              <div class="item-wrapper">
+                <div class="s-portfolio-image">
+                   <img src="/img/korean.png" alt="Korean Restaurant Web Design Example">
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Korean beef</strong></p>
+              <p>Webdesign, brand & logotypes</p>
+            </figcaption>
+          </figure>
+          <figure class="l-flex-item f-small-line-height">
+            <div class="first-level-hover-wrapper">
+              <div class="item-wrapper">
+                <div class="s-portfolio-image">
+                   <img src="/img/facebook_thumbs.png" alt="Facebook Icon Design Example">
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Facebook thumbs</strong></p>
+              <p>Icons design, brand & logotypes</p>
+            </figcaption>
+          </figure>
+        </div>
+      </article>
+      <article class="b-black" id="about">
+        <h2 class="f-bold f-large">ABOUT</h2>
+        <section class="l-flex-container">
+          <p class="l-flex-item l-about-txt">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+          <p class="l-flex-item l-about-txt">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+        </section>
+        <section>
+          <h3 class="f-bold h3-lines">SKILLS</h3>
+          <p class="s-progress-paragraph">PHOTOSHOP</p><progress min="0" max="100" value="80"></progress></p>
+          <p class="s-progress-paragraph">HTML/CSS</p><progress min="0" max="100" value="90"></progress></p>
+          <p class="s-progress-paragraph">ILLUSTRATOR</p><progress min="0" max="100" value="70"></progress></p>
+          <p class="s-progress-paragraph">PHP</p><progress min="0" max="100" value="60"></progress></p>
+          <p class="s-progress-paragraph">WORDPRESS</p><progress min="0" max="100" value="95"></progress></p>
+        </section>
+        <section>
+          <h4 class="f-light s-progress-question-text">Do you want be one of Avengers? Join us!</h4>
+          <a href=""><button class="btn contact-btn">CONTACT US</button></a>
```

Comment:
Also, there's really no need to hard-code this text as UPPERCASE. It's a good practice to write your text out in sentence case, as you can easily transform it (via CSS) to upper, lower, or even title case, but sentence case is nearly impossible to get without running it through JavaScript.